src/character/character-sheet/edit/EditResources.tsx

```diff
@@ -105,62 +156,132 @@ export const EditResources = () => {
-            onClick={() => updateFatigue(0)}
-          />
-          <ResourceCounter
-            total={maxStamina}
-            remaining={maxStamina - fatigue}
-            color={Theme.Stamina}
-            onResourceSegmentClick={updateFatigue}
-          />
-        </ButtonCounterWrapper>
-      </ResourceSegment>
-      {showPowerPoints ? (
+        ) : null}
         <ResourceSegment>
-          Power Points:
+          Health
           <ButtonCounterWrapper>
             <Button
               text="X"
-              customcolor={Theme.PowerPoints}
-              onClick={() => updatePowerPoints(0)}
+              customcolor={Theme.Health}
+              onClick={() => updateWounds(0)}
             />
             <ResourceCounter
-              total={maxPowerPoints}
-              remaining={currentPowerPoints}
-              color={Theme.PowerPoints}
-              onResourceSegmentClick={updatePowerPoints}
+              total={maxHealth}
+              remaining={maxHealth - wounds}
+              color={Theme.Health}
+              onResourceSegmentClick={updateWounds}
             />
           </ButtonCounterWrapper>
         </ResourceSegment>
-      ) : null}
-      <IconButton
-        icon={Icons.Campfire}
-        viewbox="0 0 1080 1296"
-        onClick={fullRest}
-        text="Full Rest"
-      ></IconButton>
+        <ResourceSegment>
+          Stamina
+          <ButtonCounterWrapper>
+            <Button
+              text="X"
+              customcolor={Theme.Stamina}
+              onClick={() => updateFatigue(0)}
+            />
+            <ResourceCounter
+              total={maxStamina}
+              remaining={maxStamina - fatigue}
+              color={Theme.Stamina}
+              onResourceSegmentClick={updateFatigue}
+            />
+          </ButtonCounterWrapper>
+        </ResourceSegment>
+        {showPowerPoints ? (
+          <ResourceSegment>
+            Power Points:
+            <ButtonCounterWrapper>
+              <Button
+                text="X"
+                customcolor={Theme.PowerPoints}
+                onClick={() => updatePowerPoints(0)}
+              />
+              <ResourceCounter
+                total={maxPowerPoints}
+                remaining={currentPowerPoints}
+                color={Theme.PowerPoints}
+                onResourceSegmentClick={updatePowerPoints}
+              />
+            </ButtonCounterWrapper>
+          </ResourceSegment>
+        ) : null}
+        <IconButton
+          icon={Icons.Campfire}
+          viewbox="0 0 1080 1296"
+          onClick={fullRest}
+          text="Full Rest"
+        />
+        <ResourceSegment>
+          <FlexRow>
+            <TextElement>Temporary Effects</TextElement>
+            <IconButton
+              onClick={() => setShowAddBuffModal(true)}
+              icon={Icons.Add}
+              text={"Add"}
+            />
+          </FlexRow>
+          {Boolean(temporaryEffects) ? (
```

Comment:
Boolean is not needed
```suggestion
          {temporaryEffects ? (
```